packages/wd-base/src/demo.scss

```diff
@@ -0,0 +1,166 @@
+@import "colors";
+@import "typography";
+
+/* ==========================================================================
+   Demo Page
+   ========================================================================== */
+
+/* Reset
+   ========================================================================== */
+* {
```

Comment:
Is that completely effective? or should we use something like https://cssreset.com/scripts/eric-meyer-reset-css/ ?

Reply:
@pragmaticivan It works for the purposes of styling the demo pages right now - I added it in order to have a blank slate. I imagine if we want a more detailed reset we can add one easily - what do you think?

Reply:
Sounds good to me.